src/model/object/entity/individual/piercy/piercy.h

```diff
@@ -27,6 +27,9 @@ class Piercy : public Individual {
      * @param energy is its life source */
     Piercy(long ID, Point position, double a, double energy);
 
+    /** Methods giving birth to an offspring */
```

Comment:
please avoid repeating the docs, I think the IDE should inherit it from the super method automatically

Reply:
not sure, I'll check up on this

Reply:
but since you can call methods of parents, it might make sense to have different documentation

Reply:
but either do the same ^^

Reply:
in this case, yes
but not in general